ember_debug/main.js

```diff
@@ -12,13 +12,15 @@ import DeprecationDebug from "ember-debug/deprecation-debug";
 import Session from "ember-debug/services/session";
 
 const Ember = window.Ember;
-const { Object: EmberObject, run, Application, namespaces } = Ember;
+const { Object: EmberObject, run, Application, Namespace } = Ember;
 
 const EmberDebug = EmberObject.extend({
 
   application: null,
   started: false,
 
+  // Using object shorthand syntax here is somehow having strange side effects.
+  // eslint-disable-next-line object-shorthand
   Port: Port,
```

Comment:
What were the side effects, `Port` is `undefined`? My first guess is because [the es6 module is being transpiled before Babel](https://github.com/emberjs/ember-inspector/blob/930fc378f232f85ab51c906a74556aab4ca17f46/ember-cli-build.js#L109-L115) so `Port` import is being transpiled to `Port['default']` before Babel gets the file.